gemfiles/ruby_2.2.10_contrib.gemfile.lock

```diff
@@ -1338,7 +1338,7 @@ GEM
     rake-compiler (1.2.1)
       rake
     redcarpet (3.5.1)
-    redis (4.1.1)
+    redis (3.3.5)
```

Comment:
Any reason not to put `redis` `< 4` as a constraint in the 2.2 section of `Appraisals`? Otherwise it risks breaking if someone runs an update.

Reply:
I got this error, it seems to me resque is using a wrong version of redis. I guess these versions of gems are so old that probably not maintained hence the dependencies could be prone the error.

```
Failure/Error: ::Resque.redis = url

NoMethodError:
  undefined method `connect' for Redis:Class
Shared Example Group: "job execution tracing" called from ./spec/datadog/tracing/contrib/resque/instrumentation_spec.rb:200
/usr/local/bundle/gems/resque-1.27.4/lib/resque.rb:116:in `redis='
./spec/datadog/tracing/contrib/resque/instrumentation_spec.rb:44:in `block (2 levels) in <top (required)>'
./spec/datadog/tracing/contrib/resque/instrumentation_spec.rb:56:in `block (2 levels) in <top (required)>'
./spec/datadog/tracing/contrib/support/tracer_helpers.rb:96:in `block (2 levels) in <module:TracerHelpers>'
./spec/spec_helper.rb:220:in `block (2 levels) in <top (required)>'
./spec/spec_helper.rb:112:in `block (2 levels) in <top (required)>'
```